fe-ionic/init.sh

```diff
@@ -18,11 +18,58 @@ while [[ "$#" > 0 ]]; do case $1 in
 esac; shift; done
 
 echo "generate project"
+# create temp-dir since non-interactive ionic-cli can not generate into existing directories
 mkdir start_$COMPONENT
 cd start_$COMPONENT
-ionic start $COMPONENT blank --type=angular --no-deps --no-git
+
+# creating project
+ionic start $COMPONENT blank --type=angular --no-deps --no-git --no-interactive
```

Comment:
``--no-interactive`` is not needed. Since ``CI=true`` it has the same behavior.

Reply:
Idea behind is to have same behaviour reproducible in your local system without having anything to remember. But can remove if wanted...